pkgs/top-level/nim-packages.nix

```diff
@@ -58,6 +58,8 @@ lib.makeScope newScope (self:
 
     regex = callPackage ../development/nim-packages/regex { };
 
+    result = callPackage ../development/nim-packages/nim-result { };
```

Comment:
This patch is out of place.

When you include an expression in Nixpkgs, you include a file for it (in the case, `development/nim-packages/nim-result`), and also the handle (the line above) in _the same Git commit_.

Please split this commit and fix the others accordingly.

Reply:
Indeed, do like you did in osh-tool itself.

Reply:
Thanks for the feedback, i split the patch in a right commit.